src/utils/ImmutableUtil.js

```diff
@@ -48,6 +30,14 @@ function immutable(store) {
 
     getState(currentState) {
       return currentState
+    },
+
+    onSerialize: (state) => {
```

Comment:
either lets make these all arrow functions or lets keep them as method shorthand.

I vote for arrow:

```js
getState: state => state,
onSerialize: state => state.toJS(),
...
```